src/components/Requests/RequestTabs.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import CodeBlock from "@theme/CodeBlock";
+
+export default function RequestTabs(props) {
+  const snippets = [
+    {
+      key: "bash",
+      component: require(`../../snippets/curl/${props.endpoint}/_${props.request}.mdx`),
+      value: "cURL",
+    },
+    {
+      key: "node",
+      component: require(`../../snippets/node/${props.endpoint}/_${props.request}.mdx`),
+      value: "Node",
+    },
+    {
+      key: "ruby",
+      component: require(`../../snippets/ruby/${props.endpoint}/_${props.request}.mdx`),
+      value: "Ruby",
+    },
+    {
+      key: "go",
+      component: require(`../../snippets/go/${props.endpoint}/_${props.request}.mdx`),
+      value: "Go",
+    },
+    {
+      key: "php",
+      component: require(`../../snippets/php/${props.endpoint}/_${props.request}.mdx`),
+      value: "PHP",
+    },
+    {
+      key: "python",
+      component: require(`../../snippets/python/${props.endpoint}/_${props.request}.mdx`),
+      value: "Python",
+    },
+  ];
+
+  return (
+    <>
+      <Tabs groupId="programming-language">
```

Comment:
note: this `groupId` is used to keep the same selected language in each component so it's possible to navigate into other requests without the need to pick a language each time